src/VCRCleanerEventSubscriber.php

```diff
@@ -98,6 +116,32 @@ private function sanitizeRequestBody(Request $request)
         $request->setBody($body);
     }
 
+    private function sanitizeResponseHeaders(Response &$response)
+    {
+        $options = RelaxedRequestMatcher::getConfigurationOptions();
+        $workspace = $response->toArray();
+
+        foreach ($options['ignoreResponseHeaders'] as $headerToIgnore) {
```

Comment:
The whole `$options` thing needs to have helper functions since this project is growing and more configuration options are being added. No way I'm going to remember each key name in a few hours.